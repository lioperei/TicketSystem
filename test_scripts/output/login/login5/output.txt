Welcome to the Banking System
Enter username
Enter Command:
	Create
	Delete
	Buy
	Sell
	Refund
	Add Credit
	Logout
Invalid
Enter Command:
	Create
	Delete
	Buy
	Sell
	Refund
	Add Credit
	Logout
