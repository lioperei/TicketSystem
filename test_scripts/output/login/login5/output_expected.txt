Welcome to the Banking System
login
Enter username
admin
Enter Command:
	Create
	Delete
	Buy
	Sell
	Refund
	Add Credit
	Logout
login
Invalid
Enter Command:
	Create
	Delete
	Buy
	Sell
	Refund
	Add Credit
	Logout
